stdlib/public/Python/Python.swift

```diff
@@ -39,12 +39,17 @@ typealias OwnedPyObjectPointer = PyObjectPointer
 final class PyReference {
   private var pointer: OwnedPyObjectPointer
 
-  init(owning pointer: OwnedPyObjectPointer) {
+  // This `PyReference`, once deleted, will make no delta change to the
+  // python object's reference count. It will however, retain the reference for
+  // the lifespan of this object.
```

Comment:
Use same doc comment as corresponding `PythonObject` initializer:
```suggestion
  /// Creates a new instance and a new reference.
```

Reply:
Oops, the corresponding `PythonObject` initializer doc comment is not appropriate.